frontend/src/features/SideWindow/beacon_malfunctions/BeaconMalfunctionDetailsFollowUp.tsx

```diff
@@ -29,9 +30,14 @@ export function BeaconMalfunctionDetailsFollowUp({ beaconMalfunctionWithDetails,
   const [today, setToday] = useState('')
   const [yesterday, setYesterday] = useState('')
   const scrollToRef = useRef<HTMLDivElement>(null)
+  const scrollEventRef = useRef<HTMLDivElement>(null)
+  const isScrollBlocked = useRef(false)
   const [comment, setComment] = useState('')
   const textareaRef = useRef<HTMLTextAreaElement>(null)
   const [textareaHeight, setTextareaHeight] = useState(50)
+  useListenForScroll(() => {
```

Comment:
Just a taste preference: I generally like to group side effects just before the `return`, both `useEffect()` and hooks that act like one (i.e.: https://github.com/MTES-MCT/monitor-ui/blob/main/src/fields/DatePicker/index.tsx#L208).